lib/src/model/hms_local_peer.dart

```diff
@@ -23,7 +24,9 @@ class HMSLocalPeer extends HMSPeer {
       HMSLocalAudioTrack? audioTrack,
       HMSLocalVideoTrack? videoTrack,
       List<HMSTrack>? auxiliaryTracks,
-      HMSNetworkQuality? networkQuality})
+      HMSNetworkQuality? networkQuality,
```

Comment:
Since we are modifying HMSPeer class, add inline comments explaining each property